### DISCUSSION TITLE
How to replace type startstop_dimming?

### DISCUSSION THREAD

**Discussion by buergi** [XKNX/xknx#1542] [Q&A]
Since many years I use light switches which use DPT3 type for dimming, at first I used them with a generic binary type, which was considered discouraged and removed. I then implemented the DPT3 types (`startstop_*`, `stepwise_*`) which were now refactored into `DPTControlDimming` (`control_dimming`), but I can't find anything about it in Home Assistant's knx integration documentation.
How do I use them? I currently have a sensor of type `startstop_dimming` which is handled by an appdaemon app.

**Comment by farmio** [comment]
Yes, the documentation for the new DPTs isn't done yet 😬

I'd recommend to not use a sensor, but a `knx.telegram` trigger for those (momentary, stateless) things. The new DPT definition will be used there if you uploaded your project file and it included those GAs. (Alternatively set the decoder manually for the trigger). https://www.home-assistant.io/integrations/knx/#telegram-trigger

Or use one of the Knx dimming blueprints you'll find in the HA community forum (or this, which is updated to use the new telegram trigger, but not that feature rich as some others https://gist.github.com/farmio/bc23cc2a1a81408751947309408ff164 )

I'm not familiar with Appdeamon, but if you do t have access to triggers, maybe you can use `knx_event` there, if you want to continue using it.

**Comment by buergi** [comment]
Thanks, your blueprint works, I still have some minor issues, but it gives me a good inspiration.
